src/main/java/nl/tudelft/ti2206/group9/util/SaveGameParser.java

```diff
@@ -0,0 +1,81 @@
+package nl.tudelft.ti2206.group9.util;
+
+import nl.tudelft.ti2206.group9.level.State;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * @author Mathias
+ */
+public class SaveGameParser {
+
+	/** Playername. */
+	private static String playername;
+	/** Number of coins.*/
+	private static int coins;
+	/** Players highscore. */
+	private static double highScore;
+	/** Number of coins collected in players last highscore run. */
+	private static int highCoins;
+	/** Boolean to indicate whether the sound is enabled. */
+	private static boolean soundEnabled;
+
+	/**
+	 * Load all data from the given file and save it in State.
+	 * @param filePath the path to the file to be read.
+	 */
+	public static final void loadGame(final String filePath) {
+		try {
+			JSONObject mainObject = parserInit(filePath);
+			playername = (String) mainObject.get("playername");
+			coins = Integer.valueOf((String) mainObject.get("coins"));
```

Comment:
*Auto-generated comment*

    Boxing/unboxing to parse a primitive nl.tudelft.ti2206.group9.util.SaveGameParser.loadGame(String)